test/string_from_type.hpp

```diff
@@ -47,7 +47,8 @@ template <> struct string_from_type<int>
 template <> struct string_from_type<long>
 { static std::string name() { return "l"; }  };
 
-template <> struct string_from_type<boost::multiprecision::cpp_bin_float_100>
+template <typename Backend>
+struct string_from_type<boost::multiprecision::number<Backend>>
```

Comment:
`Backend` can define various floating point and integral types. Is this specialization correct for all of them or should it be narrowed to floating point types only?

Reply:
Yes, it's for debug. It's used for display on screen and in svg filenames. It's OK if they all are indicated with `m`